data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+	explode "nuke explosion" 3
+	"final explode" "nuke explosion" 3
+	"final explode" "nuke residue fast" 90
+
+	description "This was once an Avgi sublight starship, one of the last models built before the discovery of the hyperdrive. Most starships around this period used ram augmented fusion drives to reach cruising speeds of up to 0.25c, but the Tachytia class was designed to reach a velocity of 0.4c with an antimatter fuel source to ignite propellant gathered with its huge ramscoop."
+	description "This particular ship was never launched on its mission, commissioned just before the hyperdrive made it obsolete. Now, it has been extensively modified by removing many of its propellant tanks, replacing them with a hyperdrive, additional equipment spaces, and several docking bays. The cryosleep bay has also been substituted for spacious habitatation cylinders, making this a true city ship."
+
+
+
+# Interceptors
+
+ship "Tremoros"
+	plural "Tremoros"
+	sprite "ship/avgi tremoros"
+	thumbnail "thumbnail/avgi tremoros"
+	attributes
+		category "Interceptor"
+		licenses
+			"Avgi Civilian"
+		"cost" 490000
+		"shields" 600
+		"hull" 3600
+		"automaton" 1
+		"crew equivalent" 1
+		"mass" 45
+		"drag" .3
+		"heat dissipation" 0.9
+		"fuel capacity" 900
+		"cargo space" 12
+		"outfit space" 210
+		"weapon capacity" 60
+		"engine capacity" 54
+		"anchor point" 1
+		weapon
+			"blast radius" 90
+			"shield damage" 720
+			"hull damage" 360
+			"hit force" 720
+	outfits
+		"Speckle Railgun" 4
+		"Speck Round" 14400
+		
+		"Lantern Fission Core"
+		"Ultracapacitor Cell" 3
+		"Diffuse Deflector"
+		"Multispectral Scanner"
+		
+		"Z-360 Fusion Torch"
+		"XY-180 Radial Thrusters"
+		"Hyperdrive"
+
+	gun -6.5 -18 "Speckle Railgun"
+	gun 6.5 -18 "Speckle Railgun"
+	gun -6.5 -18 "Speckle Railgun"
+	gun 6.5 -18 "Speckle Railgun"
+	turret 0 39
+
+	engine 0 51
+	"reverse engine" -3 -33
+		angle 30
+	"reverse engine" 3 -33
+		angle -30
+	"steering engine" -3 -33
+		angle 120
+		right
+	"steering engine" 3 -33
+		angle -120
+		left
+	"steering engine" 0 51
+		angle -30
+		right
+	"steering engine" 0 51
+		angle 30
+		left
+
+	leak "flame" 30 90
+	explode "tiny explosion" 12
+	explode "small explosion" 9
+	"final explode" "final explosion small"
+
+	description "The Tremoros-class Interceptor is a small autonomous ship designed to support patrol and interdiction missions. With an emphasis on speed, it can rocket past most adversaries even as it brings its guns to bear, making it exceedingly difficult to hit."
+	description "The Tremoros has also seen use as an escort for smaller merchant convoys, with the ability to intercept any enemies that might pose a threat before they can come within range of vulnerable civilian ships."
+
+
+# Light Warships
+
+ship "Prismaios"
+	plural "Prismaios"
+	sprite "ship/avgi prismaios"
+	thumbnail "thumbnail/avgi prismaios"
+	swizzle 0
+	attributes
+		category "Light Warship"
+		licenses
+			"Avgi Military"
+		"cost" 240000
```

Comment:
```suggestion
		"cost" 2640000
```
Higher than the Modified Argosy's 1.960M, but lower than the 3M-4M costs of human medium warships.